reactive/kubernetes_worker.py

```diff
@@ -436,39 +442,57 @@ def send_data():
 
 
 @when('kube-api-endpoint.available', 'kube-control.dns.available',
-      'cni.available')
+      'cni.available', 'endpoint.container-runtime.available')
```

Comment:
Will this prevent `watch_for_changes` from running on upgraded deployments that don't use a subordinate?  Or are we blocking in that case anyway?

Reply:
@johnsca I believe we want to block in this case, as we want them to deploy a docker subordinate after upgrading to 1.15 (as mentioned in the release notes).